config/initializers/inflections.rb

```diff
@@ -1,8 +1,38 @@
 # Be sure to restart your server when you modify this file.
 
+Rails.autoloaders.each do |autoloader|
+  autoloader.inflector.inflect(
+    'proxy_api' => 'ProxyAPI'
+  )
+end
+
 # Add new inflection rules using the following format
 # (all these examples are active by default):
 ActiveSupport::Inflector.inflections do |inflect|
+  inflect.acronym 'AAAA' # DNS IPv6 record
+  inflect.acronym 'AIX' # IBM AIX
+  # Requires our entire Api namespace to be rewritten
+  # inflect.acronym 'API' # Application Programmer Interface
```

Comment:
_with -Wpedantic, feel free to tell me no_
Wouldn't it be the right thing to do, while we're at it?

When we were migrating away from all the `*_core` gems on the smart proxy side, we aliased some constants to give us a bit of a leeway and to not have to change all the gems _at once_. Maybe we could do a similar thing here?